_source/_posts/2019-09-26-getting-started-with-elk.md

```diff
@@ -0,0 +1,375 @@
+To enable Zipkin tracing, add the zipkin profile to `blog-app`, `gateway-app` and `store-app` in **docker-compose/docker-compose.yml**.
+
+```yaml
+- SPRING_PROFILES_ACTIVE=prod,swagger,zipkin
+```
+You also need to rebuild the docker images with the **zipkin** profile, for `blog-app`, `store-app` and `gateway-app` with the following Maven command:
+```SHELL
+./mvnw package -Pprod -Pzipkin verify jib:dockerBuild -DskipTests
+```
+
+>ProTip:  If you're on a system with bash shell, like Linux or MacOs, you can do this from the `jhipster` folder to build each project at once:
+```
+for i in blog gateway store
+do 
+cd $i 
+./mvnw package -Pprod -Pzipkin verify jib:dockerBuild -DskipTests
+cd ..
+done
+```
+
+### Run the Monitored Microservices Architecture
+
+Are you ready for the best? Go to the `docker-compose` folder and start the services with the following command:
+
+```SHELL
+docker-compose up
+```
+
+You will see a huge amount of logging while each service starts.
+
+```
+jhipster-registry_1           | ----------------------------------------------------------
+jhipster-registry_1           | 	Application 'jhipster-registry' is running! Access URLs:
+jhipster-registry_1           | 	Local: 		http://localhost:8761
+jhipster-registry_1           | 	External: 	http://172.20.0.2:8761
+jhipster-registry_1           | 	Profile(s): 	[composite, dev, swagger, oauth2]
+jhipster-registry_1           | ----------------------------------------------------------
+```
+
+Log into the JHipster Registry at [**http://localhost:8761**](http://localhost:8761) with Okta user credentials and check the service's health.
+
+{% img blog/getting-started-with-elk/registry.png alt:"JHipster Registry" width:"800" %}{: .center-image }
+
+Once all the services are up, log into the gateway application and create some blogs and posts to generate traffic. To do this, use the **Entities** menu at the top left of the application. The gateway home is at [**http://localhost:8080**](http://localhost:8080)
+
+The fun part! Access the JHipster Console at [**http://localhost:5601**](http://localhost:5601). Go to the Dashboards section, and open the **requests-dashboard**. You should see some nice curves:
+
+{% img blog/getting-started-with-elk/requests-dashboard.png alt:"Requests Dashboard" width:"800" %}{: .center-image }
+
+Since we integrated the JHipster Console with Zipkin UI, in the traces-dashboard, you find the longest traces duration on the left. If you click on a traceId on the right, it will open the trace in the UI and you will be able to examine the flow.
```

Comment:
```suggestion
Since you integrated the JHipster Console with Zipkin UI, in the traces-dashboard, you can find the longest traces duration on the left. If you click on a traceId on the right, it will open the trace in the UI and you will be able to examine the flow.
```